src/api.js

```diff
@@ -10,12 +10,22 @@ export const getTopics = () => {
     })
 };
 
-export const getArticles = (topic) => {
+export const getArticles = (sort_by,order,topic) => {
```

Comment:
We could refactor this to be a lot tidier using axios options. Have a look at the docs to see how we can build params into the url query conditionally.